src/netius/base/legacy.py

```
@@ -336,14 +336,18 @@ def values(associative):
 
 def xrange(start, stop=None, step=1):
     if PYTHON_3:
-        return _range(start, stop, step) if stop else _range(start)
-    return _xrange(start, stop, step) if stop else _range(start)
+        return _range(start, stop, step) if not stop == None else _range(start)
+    return _xrange(start, stop, step) if not stop == None else _range(start)
```

Ruff (0.16.2)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)